src/core/MOM.F90

```diff
@@ -191,7 +200,8 @@ module MOM
                                      !! set by calling the function useRegridding() from the
                                      !! MOM_regridding module.
   logical :: do_dynamics             !< If false, does not call step_MOM_dyn_*. This is an
-                                     !! undocumented run-time flag that is fragile.
+  logical :: do_online               !< If false, does not call step_MOM_dyn_*. This is an
```

Comment:
This line is a holdover from before I separated out the main offline subroutine  step_tracers. As it stands, this variable should not exist. I'll run through and double check that all references to do_online are removed from step_MOM and delete this declaration,